modules/core/src/de/diedavids/cuba/dataimport/binding/DatatypeFactory.groovy

```diff
@@ -75,15 +76,23 @@ class DatatypeFactory {
     }
 
     @SuppressWarnings('SimpleDateFormatMissingLocale')
-    private Date getDateValue(ImportConfiguration importConfiguration, String rawValue) {
-        SimpleDateFormat formatter = new SimpleDateFormat(importConfiguration.dateFormat)
-        formatter.parse(rawValue)
+    private Date getDateValue(ImportConfiguration importConfiguration, String rawValue, DataRow dataRow) {
+        try {
+            SimpleDateFormat formatter = new SimpleDateFormat(importConfiguration.dateFormat)
+            formatter.parse(rawValue)
+        } catch (ParseException e) {
+            log.warn("Date could not be read: '$rawValue' in [$dataRow]. Will be ignored.")
```

Comment:
```suggestion
            log.warn("Date could not be read: '$rawValue' in [$dataRow], because it does not match the configured date format: '${importConfiguration.dateFormat}'. Will be ignored.")
```